sql/parser/builtins.go

```diff
@@ -106,39 +109,39 @@ type builtin struct {
 	fn     func(EvalContext, DTuple) (Datum, error)
 }
 
-var timestampMinusBinOp = binOps[binArgs{Minus, timestampType, timestampType}]
+var timestampMinusBinOp = BinOps[BinArgs{Minus, timestampType, timestampType}]
 
-// The map from function name to function data. Keep the list of functions
-// sorted please.
-var builtins = map[string][]builtin{
+// Builtins map from function name to function data. Keep the list of
```

Comment:
I'd remove the bit about sortedness in a doc comment. Also "Builtins contains the built-in functions indexed by name"?